src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery loterry = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
Magic number `3` should be constant variable

Reply:
+